partiql-tests-data/eval/query/select/group-by.ion

```diff
@@ -0,0 +1,1092 @@
+'simple-group-by'::[
```

Comment:
opinion, but this may belong in `query/group_by` or something similar

Reply:
Agreed, will change.